docker-compose.yml

```diff
@@ -155,9 +155,27 @@ services:
     ##############################
     #   Long running processes   #
     ##############################
-    php-connect-four-follow-event-store:
+    php-connect-four-follow-event-store-1:
         <<: *php-container
-        command: bin/restartOnChange bin/console connect-four:follow-event-store pointer --select-all-subscribers -v
+        command: bin/restartOnChange bin/console connect-four:follow-event-store pointer-1 --select-all-subscribers -v
+    php-connect-four-follow-event-store-2:
+        <<: *php-container
+        command: bin/restartOnChange bin/console connect-four:follow-event-store pointer-2 --select-all-subscribers -v
+        environment:
+            WAIT_FOR: mysql:3306,redis:6379,rabbit-mq:5672,nchan:81
```

Comment:
Those environment variables should be squashed. Before that pull requests, they've been in one place. This duplication needs to be resolved.